tests/test_matrix_utils_invite.py

```
@@ -382,3 +377,95 @@ async def test_on_invite_with_id_in_mapping(mock_logger: MagicMock) -> None:
         "Room '!abc123:matrix.org' is in matrix_rooms configuration, accepting invite"
     )
     mock_client.join.assert_called_once_with("!abc123:matrix.org")
+
+
+@pytest.mark.usefixtures("reset_matrix_utils_globals")
+@patch("mmrelay.matrix_utils.logger")
+async def test_on_invite_canonical_alias_match(mock_logger: MagicMock) -> None:
```

Ruff (0.15.9)

[warning] Unused function argument: `mock_logger`

(ARG001)